### ISSUE TITLE
RightButtons jumps on Android

### ISSUE THREAD

**Issue by idutka** [wix/react-native-navigation#7832]
### What happened?

When the icon is used for rightButtons inside topBar, it jumps when the new screen is pushed/popped in the stack.

![Jan-22-2024 16-18-56](https://github.com/wix/react-native-navigation/assets/866748/23175bd0-c3cd-4f7b-b852-403de946054a)

Note: If a custom component is used for the button, the button doesn't jump.

### What was the expected behaviour?

The RightButton with the icon should not jump during navigation.

Is there any solution for this other than using a custom button? A custom button adds unnecessary complexity.

### Was it tested on latest react-native-navigation?

- [X] I have tested this issue on the latest react-native-navigation release and it still reproduces.

### Help us reproduce this issue!

It can be reproduced with Playground app. Just add the icon and use Push navigation.

``` javascript
export default class PushedScreen extends NavigationComponent<Props> {
  static options(): Options {
    return {
      topBar: {
        testID: PUSHED_SCREEN_HEADER,
        title: {
          text: 'Pushed Screen',
        },
        rightButtons: [
          {
            id: 'singleBtn',
            text: 'single',
            testID: TOP_BAR_BTN,
            icon: require('../../img/navicon_edit.png'), // Add icon
          },
        ],
        backButton: {
          testID: BACK_BUTTON,
          enableMenu: false,
        },
      },
    };
  }
...
```

### In what environment did this happen?

React Native Navigation version: 
React Native version: 
Has Fabric (React Native's new rendering system) enabled: (yes/no) 
Node version: 
Device model: 
Android version:

**Comment by d4vidi** [comment]
@idutka the issue seems easy to reproduce. I'm working on tracking down the source - seems I have a lead.

**Comment by d4vidi** [comment]
Fix released in [v7.38.4](https://github.com/wix/react-native-navigation/releases/tag/7.38.4)
